ThunderCloud/DeveloperModeController.swift

```diff
@@ -127,16 +127,17 @@ public class DeveloperModeController: NSObject {
         progressHandler?(.preparing, 0, 0, nil)
         ContentController.shared.cleanoutCache()
         progressHandler?(.downloading, 0, 0, nil)
-        ContentController.shared.downloadUpdatePackage(fromURL: "\(apiBaseURL)/\(apiVersion)/apps/\(appId)/bundle?density=x2&environment=test") { [weak self] (stage, downloaded, totalSize, error) -> (Void) in
-            
-            if let progressHandler = progressHandler {
-                progressHandler(stage, downloaded, totalSize, error)
-            }
-            
-            if stage == .finished {
-                DeveloperModeController.appIsInDevMode = true
-                self?.finishSwitching()
-            }
+        if let _deltaDirectory = ContentController.shared.deltaDirectory {
```

Comment:
Wondering if it'd be nicer to have this as a guard instead, just to reduce the levels of indentation going on. What do you think? :)